lisa/sut_orchestrator/libvirt/transformers.py

```diff
@@ -211,14 +212,30 @@ def install(self) -> str:
         packages_list = self._distro_package_mapping[type(linux).__name__]
         self._log.info(f"installing packages: {packages_list}")
         linux.install_packages(list(packages_list))
+        self._fix_mariner_installation()
         return self._get_version()
 
+    # Some fixes to the libvirt installation on Mariner.
+    # Can be removed once the issues have been addressed in Mariner.
+    def _fix_mariner_installation(self) -> None:
+        if not isinstance(self._node.os, CBLMariner):
+            return
+
+        username = self._node.tools[Whoami].get_username()
+        self._node.execute(f"usermod -aG libvirt {username}", shell=True, sudo=True)
```

Comment:
This can be moved to a tool

Reply:
Also we can wrap getting `username` in the usermod call

Reply:
Done.